requirements.txt

```
@@ -1 +1 @@
-pika>=0.11.0,<0.12.0
+pika==1.3.1
```

Let's use the https://peps.python.org/pep-0440/#version-specifiers version specifier to allow pika to upgrade to a higher patch release (if any security fix comes).
```suggestion
pika~=1.3.1
```

---

Thanks for the tip, I have updated the version specifier.

---

I bumped the pack version as a patch considering there are no API or code changes.

---

Sounds good, please also update the https://github.com/StackStorm-Exchange/stackstorm-rabbitmq/blob/master/CHANGES.md

---

Sorry, forgot about the changelog! That is now updated as well